app/assets/stylesheets/components/page_specific/_glossary.scss

```diff
@@ -0,0 +1,67 @@
+.glossary__term {
+  cursor: pointer;
+
+  color: $color-green-primary;
+  border-bottom: 1px solid $color-green-primary;
+  line-height: 18px;
+  position: relative;
+}
+
+.glossary__description {
+  @include body(16, 22);
+  border: 3px solid $color-blue-bright;
+  padding: $baseline-unit*2;
+  display: none;
+  position: absolute;
+  z-index: 2;
+  background: #fff;
+  width: 90%;
+  font-weight: 100;
+  left:auto;
+}
+
+.glossary__description--active {
+  display: block;
+}
+
+.glossary__description-title {
+  @include body(22, 30);
+  display: block;
+  font-weight: 700;
+  text-transform: capitalize;
+  color: $color-green-primary;
+  margin-top: $baseline-unit;
+  margin-bottom: $baseline-unit;
+  padding-bottom: $baseline-unit;
+}
+
+.glossary__description-content {
+  margin-bottom: $baseline-unit*10;
+}
+
+.glossary__description-close {
+  @include body(14, 16);
+  display: block;
+  position:absolute;
+  top:-15px;
+  right:-15px;
+  background-color: white;
+  border: 2px solid $color-grey-primary;
+  border-radius: 50%;
+  width: 30px;
+  height: 30px;
+
+  svg {
+    position: absolute;
+    top: 5px;
+    left: 5px;
+    width: 16px;
+    height: 16px;
+  }
+
+}
+
+  sub {
+    @include body(14, 16);
+    vertical-align: super;
+  }
```

Comment:
The indentation is slightly off here, its indented as if its nested but isn't.  This should probably be nested within .glossary__term:
```
.glossary__term {
  sub {
  }
}
```
otherwise it will apply to all sub tags.